src/hotspot/cpu/aarch64/pauth_aarch64.hpp

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021, Arm Limited. All rights reserved.
```

Comment:
This file is not a derived work of Oracle's, as far as I can see.

Reply:
As I understood it, as stated in the Oracle Contributor Agreement, we share copyright with Oracle.  And as far as I can tell, Arm has always added new files with both an Arm and Oracle line (most obvious example I can find of this is cpu/aarch64/registerMap_aarch64.cpp, which has a single log entry from Nick).

Reply:
OK. Your call, but you don't need to do that.